### DISCUSSION TITLE
Agenda for the Flatcar community call on Tuesday, 11th of January 5:30pm CET / 10pm IST / 4:30pm GMT / 11:30am EST / 8:30am PST

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#594] [Flatcar Office Hours]
# Agenda for the Flatcar community call on Tuesday, 11th of January 5:30pm CET / 10pm IST / 4:30pm GMT / 11:30am EST / 8:30am PST

- Zoom link: [https://us06web.zoom.us/j/84336116610](https://us06web.zoom.us/j/84336116610)
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: [link](https://www.youtube.com/watch?v=X_nqgXLOmLk)
- Slide deck <>
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro / check-in of all participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- 5.15 Kernel in Alpha 
- upcoming FOSDEM devroom talks overview

## Spotlight
- 2021 recap - cgroupsv2, systemd updates, docker 20, ARM64  
- Flatcar on Raspberry Pi blog
- Fleetlock (@aniruddha2000, @tormath1 )

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/5)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by t-lo** [comment]
Podman support is blocked by https://github.com/flatcar-linux/Flatcar/issues/447, the PR has been dormant for some time and we should probably close it. So I wouldn't bring it up in the community call.

We could point out our upcoming talks at FOSDEM though, I'll work out a list of what has been accepted.

**Reply by miao0miao** [reply]
@t-lo please feel free to edit the agenda!

**Reply by t-lo** [reply]
Thanks, updated.

**Comment by tormath1** [comment]
If @aniruddha2000 is available, we'd like to showcase FleetLock + Flatcar (it was initially scheduled for the community call of Dec.) :rocket:

**Reply by aniruddha2000** [reply]
Yes I am available. We can showcase our work :D

**Comment by t-lo** [comment]
Agenda + slides available with https://github.com/flatcar-linux/Flatcar/pull/610.
